Enterprise-CMCS/macpro-mdct-mfp
# SAR Initiative Dashboard Page

### Description

With all the changes we've made, the SAR.json isn't displaying the initiative dashboard page correctly. The json just needed some light updating and the page is now showing correctly.

![Screenshot 2023-12-11 at 4 26 32 PM](https://github.com/Enterprise-CMCS/macpro-mdct-mfp/assets/127151429/3f1ec6ab-ada3-4a2a-921e-c88bbb4010cd)

P.S. I tossed in a fix for the quarter logic not generating correctly, I forgot it goes up to 4 and not 2.

### Related ticket(s)


CMDCT-33

---
### How to test

1) Sign into MFP, any state user
2) Create a WP
3) Go to State- or Territory-Specific Initiatives 
4) Add and fill out an initiative in the WP
5) Go to the SAR dashboard and create a new SAR
6) Select to State- or Territory-Specific Initiatives
7) The initiative should have copied over from WP
8) Click the [Edit] button
9) The initiative dashboard will now display

### Important updates


N/A

---
### Author checklist


---


_convert to a different template: [test → val](?expand=1&template=test-to-val-deployment.md)_ | _[val → prod](?expand=1&template=val-to-prod-deployment.md)_
